src/app-layout/__integ__/runtime-drawers.test.ts

```diff
@@ -73,6 +73,28 @@ describe.each(['classic', 'refresh', 'refresh-toolbar'] as const)('%s', theme =>
         await expect(page.getText('[data-testid="current-size"]')).resolves.toEqual('resized: true');
       })
     );
+
+    test(
+      'should show sticky elements on scroll in drawer',
+      setupTest(async page => {
+        await page.setWindowSize(viewports.desktop);
+        await page.click(
+          createWrapper().findToggle('[data-testid="use-drawers-toggle"]').findNativeInput().toSelector()
+        );
```

Comment:
This page has known glitches when switching between tools and drawers without full page reload. 

If you need specifically the drawers list for the test case, you need to pass it as a query parameter

Reply:
It was working as expected when I toggled the drawers button, but I updated to use parameters instead for safety.